src/webextension/widgets/panel.css

```diff
@@ -13,8 +13,9 @@
   display: flex;
   align-items: center;
   font-weight: 500;
-  height: 32px;
-  padding: 2px 5px;
+  background: #ededf0;
+  height: 54px;
+  padding: 0 12px;
```

Comment:
Is there any reason for specifying the `height` and `padding like this instead of this?

```css
height: 30px;
padding: 12px;
```

That seems a little cleaner to me, but I don't feel super-strongly about this.